packages/website/src/shared/contentful.js

```diff
@@ -0,0 +1,10 @@
+import contentful from 'contentful'
+
+export const setupClient = () => {
+  const client = contentful.createClient({
+    space: '6dcasuqyhf2a',
```

Comment:
These are exposed during api calls so are not that big an issue been in the code, I can look at being able to use env files with our setup but can then replace these codes after

Reply:
Yeah, I think better to access these things through config.

We follow this pattern elsewhere - for example our stripe publishable key gets [added to the config as it's sandbox version](https://github.com/peckhamcc/www/blob/master/packages/config/index.js#L619), then accessed [via env vars in production mode](https://github.com/peckhamcc/www/blob/master/packages/config/index.js#L664) and [set at build time](https://github.com/peckhamcc/www/blob/master/packages/website/esbuild.js#L39) for the front end.

Locally your dev setup will use the sandbox value, the lambdas will use the production env var and the deployed frontend will use the build time value all from the same source file.

Reply:
ah cool, we have configs.. let me have a look how these work and move the contentful ones over to there